dev/io.openliberty.org.eclipse.microprofile.lra.1.0/bnd.bnd

```diff
@@ -0,0 +1,35 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#    IBM Corporation 
+# This wraps the external JAR.
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/rejar.props
+
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-SymbolicName: io.openliberty.org.eclipse.microprofile.lra.1.0; singleton:=true
+                     
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+# We will whittle this down once it is working
+Export-Package: \
+  org.eclipse.microprofile.lra*; version="1.0.0"
+
+# TODO what should this be?
+#WS-TraceGroup: REACTIVESTREAMS
```

Comment:
This is one place to define a trace group which the user can use to turn on trace for the whole feature rather than having to list package and class names. Elsewhere, you've used `LRA` as the tracegroup

I don't think you want this in the API bundle anyway since it won't have trace in it.